spec/features/batch_ingest/aapb_pbcore_zipped_spec.rb

```diff
@@ -5,96 +5,95 @@
+      # Instantiations only. A little odd, but this is just how the ingest works
+      # right now.
+      @pbcore_description_documents.count \
+        + instantiations.count \
+        + physical_instantiation_essence_tracks.count
+    end
 
-      it 'creates the correct number of batch item records' do
-        expect(@batch.batch_items.to_a.count).to eq expected_batch_item_count
-      end
+    it 'creates the correct number of batch item records' do
+      expect(@batch.batch_items.to_a.count).to eq expected_batch_item_count
+    end
 
-      it 'ingests the correct number of objects' do
-        expect(@ingested_objects.count).to eq expected_batch_item_count
-      end
+    it 'ingests the correct number of objects' do
+      expect(@ingested_objects.count).to eq expected_batch_item_count
+    end
 
-      it 'creates additional BatchItem records that all share the same `id_within_batch` value' do
-        batch_items_by_repo_object_id = @batch.batch_items.index_by(&:repo_object_id)
-        ingested_objects_by_id = @ingested_objects.index_by(&:id)
-        batch_items_by_repo_object_id.each do |repo_object_id, batch_item|
-          # There should be at most 1 parent work ID; assets won't have one.
-          parent_work_id = ingested_objects_by_id[repo_object_id].parent_work_ids.first
-          if parent_work_id
-            parent_batch_item = batch_items_by_repo_object_id[parent_work_id]
-            expect(batch_item.id_within_batch).to eq parent_batch_item.id_within_batch
-          end
+    it 'creates additional BatchItem records that all share the same `id_within_batch` value' do
+      batch_items_by_repo_object_id = @batch.batch_items.index_by(&:repo_object_id)
+      ingested_objects_by_id = @ingested_objects.index_by(&:id)
+      batch_items_by_repo_object_id.each do |repo_object_id, batch_item|
+        # There should be at most 1 parent work ID; assets won't have one.
+        parent_work_id = ingested_objects_by_id[repo_object_id].parent_work_ids.first
+        if parent_work_id
+          parent_batch_item = batch_items_by_repo_object_id[parent_work_id]
+          expect(batch_item.id_within_batch).to eq parent_batch_item.id_within_batch
         end
       end
+    end
 
-      it 'has a status of completed' do
-        expect(@batch.status).to eq "completed"
-      end
+    it 'has a status of completed' do
+      expect(@batch.status).to eq "completed"
+    end
 
-      it 'has no errors for any batch item' do
-        expect(@batch.batch_items.map(&:error)).to all(be_nil)
-      end
+    it 'has no errors for any batch item' do
+      expect(@batch.batch_items.map(&:error)).to all(be_nil)
+    end
 
-      it 'has status of "completed" for each batch item' do
-        expect(@batch.batch_items.map(&:status)).to all( eq 'completed' )
-      end
-    # TODO: Tests for invalid Assets.
-    else
-      skip 'Until we decide if we keep Hyrax::BatchIngest, see: https://github.com/scientist-softserv/ams/issues/58'
```

Comment:
You removed the skip here. Was that intentional?

Reply:
Yup, this [commit](https://github.com/WGBH-MLA/ams/pull/754/commits/5ac83eef15e4c5a84c7137b83b0b1b810094b197) fixed it